common/bcp47/currency.xml

```diff
@@ -277,6 +277,7 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="veb" description="Venezuelan Bolívar (1871-2008)" since="1.9"/>
             <type name="vef" description="Venezuelan Bolívar (2008-2018)"/>
             <type name="ves" description="Venezuelan Bolívar" since="34"/>
+            <type name="ved" description="Bolívar Soberano" since="41"/>
```

Comment:
The text from ISO was not in NFC:
https://r12a.github.io/app-conversion/

"Boli&#x0301;var Soberano"

Reply:
Whoops, wrong line:
"Boli\u{301}var Soberano"

Reply:
Thanks. Fixed, and also filed a ticket to improve the err message: CLDR-15386